src/parquet/file/reader.h

```diff
@@ -79,7 +79,8 @@ class PARQUET_EXPORT ParquetFileReader {
   ParquetFileReader();
   ~ParquetFileReader();
 
-  // API Convenience to open a serialized Parquet file on disk
+  // API Convenience to open a serialized Parquet file on disk, using SAMPLE
+  // implementations of RandomAccessSource
```

Comment:
It would be clearer to state that this method uses built-in IO interface implementations that were created for testing, and may not be robust to all use cases.